tests/foreman/cli/test_docker.py

```diff
@@ -1521,53 +1515,51 @@ def test_positive_upload_image(self):
             is eventually implemented
 
             compute_resource = make_compute_resource({
-                'organization-ids': [self.org['id']],
-                'provider': DOCKER_PROVIDER,
-                'url': 'http://{0}:2375'.format(self.docker_host.ip_addr),
+                'organization-ids': [module_org.id],
+                'provider': 'Docker',
+                'url': f'http://{docker_host.ip_addr}:2375',
             })
             container = make_container({
                 'compute-resource-id': compute_resource['id'],
-                'organization-ids': [self.org['id']],
+                'organization-ids': [module_org.id],
             })
             Docker.container.start({'id': container['id']})
             """
             container = {'uuid': 'stubbed test'}
             repo_name = gen_string('alphanumeric').lower()
             # Commit a new docker image and verify image was created
-            result = ssh.command(
+            result = docker_host.execute(
                 'docker commit {0} {1}/{2}:latest && '
                 'docker images --all | grep {1}/{2}'.format(
-                    container['uuid'], repo_name, REPO_UPSTREAM_NAME
-                ),
-                self.docker_host.ip_addr,
+                    container['uuid'], repo_name, DOCKER_UPSTREAM_NAME
+                )
             )
-            self.assertEqual(result.return_code, 0)
+            assert result.status == 0
             # Save the image to a tar archive
-            result = ssh.command(
-                'docker save -o {0}.tar {0}/{1}'.format(repo_name, REPO_UPSTREAM_NAME),
-                self.docker_host.ip_addr,
+            result = docker_host.execute(
+                'docker save -o {0}.tar {0}/{1}'.format(repo_name, DOCKER_UPSTREAM_NAME)
```

Comment:
F strings

Reply:
Done. The test is marked as stubbed for some reason though, so I haven't tried running the test.